### ISSUE TITLE
ngrok tunnel not work

### ISSUE THREAD

**Issue by partogikukuh** [Ignitetch/AdvPhishing#123]
any idea to fix that?

**Comment by roshanleharwani** [comment]
Have you Configured your ngrok auth token ??

**Comment by partogikukuh** [comment]
yes already

**Comment by partogikukuh** [comment]
yes already configure auth token but the page is still not working. it just
said tunnel is n not found

**Comment by roshanleharwani** [comment]
okay then 
Tell me you're using this script on your PC or your phone

**Comment by roshanleharwani** [comment]
Because on phone you need to turn on your Hotspot

**Comment by partogikukuh** [comment]
im using kali linux in termux on my phone

**Comment by partogikukuh** [comment]
i already turn my hotspot on
still get same result

**Comment by roshanleharwani** [comment]
May be then you should try to reinstall the ngrok or try to run  this script on your PC if you have any

**Comment by ramees01** [comment]
> any idea to fix that?

its the regex in the grep statement which is causing the problem

**Comment by aveenain** [comment]
`Tunnel 9.ngrok.io not found`

I got this issue when using the tools. I'm using VMWare Kali Linux version 2021.3.
Any idea how to fix this?

**Comment by Habibu-R-ahman** [comment]
I am also facing same issue in Parrot OS

**Comment by SpaceCoalescence** [comment]
The problem isnt ngrok or the os, its your connection so the solution ive used is use a vpn like protonvpn or other vpn if youd like but yea and then use ngrok. Sometimes it will say ngrok tunnel not found so you keep reconnecting or use other vpn and it will work

**Comment by offsat** [comment]
my.ngrok

**Comment by DJoeDaniel** [comment]
ngrok version 2 does not supports phishing

**Comment by Supremus** [comment]
> ngrok version 2 does not supports phishing

Then what's the solution

**Comment by mohantirumalasetti** [comment]
i just solved the issue just clone it from https://github.com/mohantirumalasetti/AdvPhishing

**Comment by Hosted-EU** [comment]
There’s A New Phishing Tool Out There.
It Can Generate URLs For Phishing Pages.
And Able To Steal Cookies from Phishing Pages To Bypass 2FA 🔥

Watch Video On How It Works : https://youtube.com/watch?v=BJv3jFuJJOw
